src/main/java/MyResource.java

```diff
@@ -12,17 +12,18 @@
 
 @Path("/api")
 public class MyResource {
```

Comment:
It could perhaps be an idea to move this class down to the "resources" namespace? In that way all your "resources" classes will be located in one place.

Reply:
moved MyResource.java to "resources" folder